packages/auth/src/grant/service.test.ts

```diff
@@ -164,7 +197,10 @@ describe('Grant Service', (): void => {
       expect(fetchedGrant?.id).toEqual(grant.id)
     })
     test('Can fetch a grant by its interaction information', async (): Promise<void> => {
-      const fetchedGrant = await grantService.getByInteraction(grant.interactId)
+      const fetchedGrant = await grantService.getByInteraction(
+        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+        grant.interactId!
```

Comment:
```typescript
export function notNullOrUndef<T>(
  t: T | null | undefined,
  name = '<unknown>'
): T | never {
  if (t == null) {
    throw new Error(`Expecting not null for ${name}`)
  } else {
    return t
  }
}
```

Found a helper like above is useful to get runtime checks as well. 

Use as:

```typescript 
notNullOrUndef(grant.interactionId, 'grant.interactionId')
```

Pity TypeScript doesn't have some kind of macros